src/library/scala/Option.scala

```diff
@@ -270,7 +270,7 @@ sealed abstract class Option[+A] extends Product with Serializable {
    *  // Returns None because the partial function doesn't cover the case.
    *  Some("ftp") collect {case "http" => "HTTP"}
    *
-   *  // Returns None because None is passed to the collect method.
+   *  // Returns None because there is no value to pass to the partial function.
```

Comment:
Suggestion, how about something like:

collect over None will always produce None regardless of the partial function supplied

?